src/code-components/RichTextArea/Editor.tsx

```diff
@@ -55,7 +55,7 @@ const Editor = forwardRef<Quill | null, EditorProps>(
         const editorHtml = quill.root.innerHTML.trim();
 
         if (value?.trim() !== editorHtml) {
-          quill.clipboard.dangerouslyPasteHTML(value || "");
+          value ? (quill.root.innerHTML = value) : (quill.root.innerHTML = "");
```

Comment:
This could be simplified I guess?

```suggestion
          quill.root.innerHTML = value ?? "";
```

Reply:
Also other solution that people used to troubleshoot this was to init the editor as readonly and then once editor was ready they called `quill.enable();`

https://github.com/zenoamaro/react-quill/issues/317